bin/start_wifi_connect.sh

```diff
@@ -3,9 +3,10 @@
 # vim: tabstop=4 shiftwidth=4 softtabstop=4
 # -*- sh-basic-offset: 4 -*-
 
-set -euox pipefail
+set -euo pipefail
 
 IS_CONNECTED=''
+VERBOSE=${VERBOSE:-true}
```

Comment:
When I copied it from balenaOS's repository &ndash; `[balena-labs-projects/wifi-connect][1]`, specifically from `[start.sh][2]` &ndash; the script defaults to the verbose mode when `VERBOSE` is not defined. The purpose of the flag is to print the `echo` statements if not set to `false`.

I'll remove the flag because it's overkill.

[1]: https://github.com/balena-labs-projects/wifi-connect
[2]: https://github.com/balena-labs-projects/wifi-connect/blob/master/start.sh